### DISCUSSION TITLE
【极简首页】不能触底加载，不能继续翻页

### DISCUSSION THREAD

**Discussion by STAOY** [the1812/Bilibili-Evolved#4428] [综合讨论 / General]
![image](https://github.com/the1812/Bilibili-Evolved/assets/86558267/c268e1cf-f965-463c-877a-d1e34cadba25)
![image](https://github.com/the1812/Bilibili-Evolved/assets/86558267/c268e1cf-f965-463c-877a-d1e34cadba25)
感谢bilibili-evloved开发，确实方便了b站内容获取
如图，我主要使用【极简首页】功能设置了4列，浏览器默认67%缩放。
没有“加载中...”字样也不能继续滚动。

**Comment by Geeyun-JY3** [comment]
相关：#3468、#4302

另外你在 #4302 下评论的
> 没有使用极简首页和清爽首页，也无法实现触底加载

是什么意思？

**Comment by the1812** [comment]
"推荐" 从来就没支持过翻页
